code/modules/power/supermatter/supermatter.dm

```diff
@@ -305,12 +305,6 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	else
 		investigate_log("has exploded.", INVESTIGATE_ENGINES)
 		explosion(get_turf(T), explosion_power * max(gasmix_power_ratio, 0.205) * 0.5 , explosion_power * max(gasmix_power_ratio, 0.205) + 2, explosion_power * max(gasmix_power_ratio, 0.205) + 4 , explosion_power * max(gasmix_power_ratio, 0.205) + 6, 1, 1)
-		if(power > POWER_PENALTY_THRESHOLD)
-			investigate_log("has spawned additional energy balls.", INVESTIGATE_ENGINES)
-			var/obj/singularity/energy_ball/E = new(T)
-			E.energy = power
-		qdel(src)
-
 //this is here to eat arguments
```

Comment:
.

Reply:
Hmm?

Reply:
Test it lol

Reply:
I'm not sure what exactly you want me to test here.

Reply:
line 312 look at what you are removing

Reply:
It's to delete the Supermatter Crystal during the tesla delam.